graph-db/serverless/src/utils/cmr/indexCampaign.js

```diff
@@ -0,0 +1,54 @@
+import gremlin from 'gremlin'
+
+const gremlinStatistics = gremlin.process.statics
+
+/**
+ * Given a project object, Gremlin connection, and associated collection, index campaign and build relationships for any GENERAL campaign that exists in common with other nodes in the graph database
+ * @param {JSON} project UMM project object
+ * @param {Connection} gremlinConnection a connection to the gremlin server
+ * @param {Graph Node} dataset the parent collection vertex in the gremlin server
+ * @returns null
+ */
+const indexCampaign = async (project, gremlinConnection, dataset, conceptId) => {
+  // CMR UMM-C actually put the campaign info in the Projects ShortName field rather than the Campaigns field for historical reasons
```

Comment:
I have confirmed this with Erich. This is something that the UMM powerhouse team will eventually fix and we will update it here when that happens.